LCD.md

```diff
@@ -13,7 +13,7 @@ The list of available LCD displays and links to LCD driver component and touch d
 | <img src="docu/pics/1.28inch-LCD-Module-1.jpg" width="150"> | [Waveshare 1.28" round LCD](https://www.waveshare.com/product/1.28inch-lcd-module.htm) | GC9A01 | [Component Manager](https://components.espressif.com/component/espressif/esp_lcd_gc9a01) | - | - | |
 | <img src="docu/pics/7inch-Capacitive-Touch-LCD-C_l.jpg" width="150"> | [Waveshare 7" LCD](https://www.waveshare.com/7inch-capacitive-touch-lcd-c.htm) | RA8875 | [Component Manager](https://components.espressif.com/component/espressif/esp_lcd_ra8875) | GT911 | [Component Manager](https://components.espressif.com/component/espressif/esp_lcd_touch_gt911) | |
 | <img src="docu/pics/Adafruit-PiTFT.jpg" width="150"> | [Adafruit PiTFT - 2.8"](https://learn.adafruit.com/adafruit-pitft-28-inch-resistive-touchscreen-display-raspberry-pi) | ILI9341 | [Component Manager](https://components.espressif.com/component/espressif/esp_lcd_ili9341) | STMPE610 | [Component Manager](https://components.espressif.com/component/espressif/esp_lcd_touch_stmpe610) | |
-| <img src="docu/pics/ESP32-S3-Parallel-TFT-with-Touch-ILI9488-1000x750.jpg" width="150"> | [Makerfabs Parallel TFT](https://www.makerfabs.com/esp32-s3-parallel-tft-with-touch-ili9488.html) | ILI9488 | **IN PLAN** (Waiting for HW) | FT6236 | **IN PLAN** (Waiting for HW) | |
+| <img src="docu/pics/ili9488-parallel.webp" width="150"> | Parallel 320x480 4" | ILI9488 | [Component Manager](https://components.espressif.com/components/atanisoft/esp_lcd_ili9488) | - | - | |
```

Comment:
it should likely be noted that ILI9488 using SPI also works.

Reply:
Added note. I should add the new column with supported interfaces. I will do it later...